src/test/java/com/amazon/sqs/javamessaging/message/SQSObjectMessageTest.java

```diff
@@ -70,20 +62,20 @@ public void testCreateMessageWithObject() throws JMSException {
      */
     @Test
     public void testObjectSerialization() throws JMSException {
-        Map<String, String> expectedPayload = new HashMap<String, String>();
+        Map<String, String> expectedPayload = new HashMap<>();
         expectedPayload.put("testKey", "testValue");
 
         SQSObjectMessage sqsObjectMessage = new SQSObjectMessage();
 
-        String serialized = sqsObjectMessage.serialize((Serializable) expectedPayload);
+        String serialized = SQSObjectMessage.serialize((Serializable) expectedPayload);
```

Comment:
Should this be
`String serialized = sqsObjectMessage.serialize((Serializable) expectedPayload);` instead?